contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -1005,6 +1005,47 @@ fooRef:
 
 Although not always necessary to help a controller identify a resource type, “group” is included to avoid ambiguity when the resource exists in multiple groups. It also provides clarity to end users and enables copy-pasting of a reference without the referenced type changing due to a different controller handling the reference.
 
+##### Kind vs. Resource
+
+A common point of confusion in object references is whether to construct
+references with a "kind" or "resource" field. Historically most object
+references in Kubernetes have used "kind". This is not as precise as "resource".
+Although each combination of "group" and "resource" must be unique within
+Kubernetes, the same is not always true for "group" and "kind". It is possible
+for multiple resources to make use of the same "kind".
+
+Typically all objects in Kubernetes have a canonical primary resource - such as
+“pods” representing the way to create and delete resources of the “Pod” schema.
+While it is possible a resource schema cannot be directly created, such as a
+“Scale” object which is only used within the “scale” subresource of a number of
+workloads, most object references address the primary resource via its schema
```

Comment:
```suggestion
workloads, most object references address the primary resource via its kind
```
For the sake of clarity, could you just add a separate sentence clarifying that "kind" is a schema, and then use "kind" in further sentences?

I think it's a little confusing to say things like "object references address the resource via it's schema": it introduces a third term, and the user may not know if it's a kubernetes-specific term or a general one (in this case it's the latter).

Reply:
I added a sentence to clarify this relationship. I'm not an expert on the terminology here, but the original uses of "schema" were added as a suggestion from @smarterclayton. I likely didn't merge that in quite as cleanly as I could have, so any suggestions are very appreciated.

Reply:
the newest changes look good. My suggestion to clarify kind vs resource is just to add a sentence at the top clarifying that "kind" refers to a schema, while resource refers to a collection of objects, sharing a schema, grouped under an API path.

Reply:
It may just be me, but I think "resource refers to a collection of objects, sharing a schema, grouped under an API path" actually confuses me more than the current text. I think this may be more accurate, but also potentially more information than is required in this context.

Reply:
sure, I can see that. There's possibly better wording here, but if I saw this PR without some of this context would imagine I'd ask myself what the difference between kind vs resource actually is. I hope there's some succinct way to clarify that without introducing a third term or overcomplicating.